internals/notifier.py

```diff
@@ -1130,8 +1169,8 @@ def send_emails(email_tasks):
       try:
         cloud_tasks_helpers.enqueue_task(
             '/tasks/outbound-email', task)
-      except e:
-        logging.exception('could not enqueue.')
+      except Exception as e:
+        logging.exception(f'could not enqueue. {e}')
```

Comment:
Doesn't `logging.exception()` automatically add exception info?

Reply:
Ah, you're right! TIL 🙂 
However, VS Code was highlighting this as syntactically problematic.
![Screenshot 2024-07-17 at 12 53 50 PM](https://github.com/user-attachments/assets/e358436b-8320-486d-9b64-f761b3d55b15)

This is not related to the actual changes in this PR however, and I'll revert things.

Reply:
I think it could be fixed as `except Exception:`'